_posts/2023-05-20-introduction-juanita.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: single
+title: "🎶 My Journey Through Scientific Python: The Lyrics 🎶
+ "
+excerpt: "Hi there, I'm Juanita. I recently joined pyOpenSci as a community lead to help with community building and outreach and this blogpost tells the story of how I got here. "
```

Comment:
This is what search engines use to post information about this post. so i suggest a bit more straight forward excerpt that helps a reader know what this content is about!
```suggestion
excerpt: "Juanita Gomez recently joined pyOpenSci as a community lead to help with community building and outreach. Learn more about Juanita's path to becoming a leader in community management in the scientific Python ecosystem. "
```